npc/merchants/shops.txt

```diff
@@ -216,8 +214,7 @@ moc_ruins,131,138,0	shop	Trader#moc2	93,748:-1
 moc_ruins,71,139,5	shop	Trader#moc3	93,2612:-1
 moc_ruins,125,135,6	shop	Trader#moc4	89,2609:-1,1516:-1,1522:-1
 moc_ruins,87,109,0	shop	Butcher#moc	58,517:-1
-moc_ruins,90,149,6	shop	Trader#moc5	99,513:-1,513:-1,513:-1,513:-1,513:-1,513:-1
-moc_ruins,118,170,4	shop	Pet Groomer#moc	125,537:-1,643:-1,10013:-1,10014:-1,554:-1,6113:-1,6114:-1,6115:-1
+moc_ruins,90,149,6	shop	Trader#moc5	99,513:-1
```

Comment:
I know the item is duplicated, but it's intended:

`trader "moc_ruins" "Trader" 4W_M_03 90 149 6 0 0
OnInit:
	sellitem Banana
	sellitem Banana
	sellitem Banana
	sellitem Banana
	sellitem Banana
	sellitem Banana
return`

Reply:
ok